website/docs/r/app.html.markdown

```diff
@@ -41,10 +41,6 @@ The following arguments are supported:
 * `buildpacks` - (Optional) Buildpack names or URLs for the application.
      Buildpacks configured externally won't be altered if this is not present.
 * `config_vars` - (Optional) Configuration variables for the application.
-     The config variables in this map are not the final set of configuration
```

Comment:
@sheax0r Are you removing this comment because the functionality has changed?

Reply:
@jmervine I'm removing it because it's incorrect; the removed text was copied from the `heroku_app` resource, where it applies - but the datasource is slightly different.

The data object exposes all config vars in its `config_vars` attribute.

The resource exposes only *certain* config vars in its `config_vars` attribute - ie, ones that the user has decided to care about. 

The datasource and the resource are necessarily different in this respect.

Reply:
Oh wait @jmervine I see what you're saying now! I removed this text from the resource, not the datasource! :D.

My bad. I will put it back.